KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -152,6 +154,10 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
     }
     
     func addMoreDetails() {
+        guard areTermsAndConditionsAccepted else {
```

Comment:
Usually the T&Cs are the last thing to tick before completing a step. I'm concerned a bit about the UX we have by forcing the user to accept the T&Cs before adding passenger details and card details.

Reply:
We will rework it depending on feedback.

Reply:
Oh, I didn't notice it before. I definitely agree with @dianakarh that this checkbox should be checked as last step and do not change current behavior with missing passenger/payments details.